charts/internal/cilium-monitoring/templates/configmap-dashboards.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.gep19Monitoring }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cilium-dashboards
+  namespace: {{ .Release.Namespace }}
+  labels:
+    dashboard.monitoring.gardener.cloud/shoot: "true"
+data:
+  cilium-agent-metrics-dashboard.json: |-
+  {{- .Files.Get "cilium-agent-metrics-dashboard.json" | nindent 4 }}
+
+  cilium-operator-metrics--dashboard.json: |-
+  {{- .Files.Get "cilium-operator-metrics-dashboard.json" | nindent 4 }}
+
+  hubble-dashboard-metrics-dashboard.json: |-
+  {{- .Files.Get "hubble-metrics-dashboard.json" | nindent 4 }}
```

Comment:
Please replace datasource `-- Grafana --` with `-- Plutono --` in all three dashboard files.

```
-        "datasource": "-- Grafana --",
+        "datasource": "-- Plutono --",
```

Otherwise there will be an ugly error shown more or less randomly complaining about the unknown datasource.